docs/en/latest/concepts/apisix_route.md

```diff
@@ -139,12 +139,46 @@ spec:
           - /*
         methods:
           - GET
-      backend:
-        serviceName: foo
-        servicePort: 80
-        resolveGranularity: service
+      backends:
+        - serviceName: foo
+          servicePort: 80
+          resolveGranularity: service
 ```
 
+Weight Based Traffic Split
+--------------------------
+
+There can more than one backend specified in one route rule,
+when multiple backends co-exist there, the traffic split based on weights
+will be applied (which actually uses the [traffic-split](http://apisix.apache.org/docs/apisix/plugins/traffic-split/) plugin in Apache APISIX).
+You can specify weight for each backend, the default weight is `100`.
+
+```yaml
+apiVersion: apisix.apache.org/v2alpha1
+kind: ApisixRoute
+metadata:
+  name: method-route
+spec:
+  http:
+    - name: method
+      match:
+        paths:
+          - /*
+        methods:
+          - GET
+      backends:
+        - serviceName: foo
+          servicePort: 80
+          weight: 100
+        - serviceName: bar
+          servicePort: 81
+          weight: 50
+```
+
+The above `ApisixRoute` has one route rule, which contains two backends `foo` and `bar`, the
+weight ratio is `100:50`, which means `2/3` requests will be sent to service `foo` and `1/3` requests
+will be proxied to serivce `bar`.
+
```

Comment:
Need example to show `match` with `vars`.

Reply:
OK, will added.

Reply:
Added.